.github/workflows/images.yaml

```diff
@@ -0,0 +1,129 @@
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/workqueue
+
+    - name: Build reqmon image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmon/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/reqmon
+
+    - name: Build t0_reqmon image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/t0_reqmon/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/t0_reqmon
+
+    - name: Build reqmgr2 image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmgr2/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/reqmgr2
+
+    - name: Build reqmgr2ms-output image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmgr2ms-output/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/reqmgr2ms-output
+
+    - name: Build reqmgr2ms-rulecleaner image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmgr2ms-rulecleaner/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/reqmgr2ms-rulecleaner
+
+    - name: Build reqmgr2ms-transferor image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmgr2ms-transferor/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/reqmgr2ms-transferor
+
+    - name: Build reqmgr2ms-monitor image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmgr2ms-monitor/Dockerfile.dist
+        sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile.dist
+        cat Dockerfile.dist
+        docker build . --tag registry.cern.ch/cmsweb/reqmgr2ms-monitor
+
+    - name: Build reqmgr2ms-unmerged image
+      run: |
+        curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/reqmgr2ms-unmerged/Dockerfile.dist
```

Comment:
For this MSUnmerged, I see that the Dockerfile actually depends on two images:
```
FROM registry.cern.ch/cmsweb/gfal:latest as gfal
FROM python:3.8-bullseye
```

can you please point to this `gfal` Dockerfile?